alignak/objects/schedulingitem.py

```diff
@@ -1547,6 +1547,9 @@ def consume_result(self, chk, notif_period, hosts,  # pylint: disable=R0915,R091
         ok_up = self.__class__.ok_up  # OK for service, UP for host
 
         # ============ MANAGE THE CHECK ============ #
+        if 'TEST_LOG_ACTIONS' in os.environ:
+            logger.warning("Got check result: %d for '%s'",
```

Comment:
This environment variable is useful without the DEBUG logs that are too verbose. You can set TEST_LOG_ACTIONS in your shell when you need to tweak the checks commands.